CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.10.2)
+
+# where to find our CMake modules
+set(CMAKE_MODULE_PATH "$ENV{CARMA_OPT_DIR}/cmake")
+# Required for std::lock and syntax
+set(CMAKE_CXX_STANDARD 17)
+
+# Needed so CMake can find the vendored PROJ4 module file. Th FindPROJ4.cmake
+# module file can be removed if we upgrade to a more recent PROJ version. See
+# https://github.com/usdot-fhwa-stol/carma-platform/issues/2139 for updates.
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+
+project(carma-geodetic
+    DESCRIPTION "CARMA geodetic library"
+    HOMEPAGE_URL https://github.com/usdot-fhwa-stol/carma-geodetic-lib
+	VERSION 0.0.1
```

Comment:
Semantic versioning guidelines recommend 0.1.0 for initial development releases. [Link](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase)

Reply:
I think it may be more important for us to version these packages in a consistent way in this case than in the standard way. We need to rework the debian package release process anyway because I think there are still some outstanding things there as well. We are already versioning the carma-time-lib this way. If we do correct this or change this, we may want to do it for all the stol debian packages at once to be consistent.

Reply:
Yeah that's fine. I don't think there are any hard rules for versioning < 1.0.0.